docs/src/running-validator/validator-start.md

```diff
@@ -430,3 +430,9 @@ which starts the solana validator process uses "exec" to do so (example: "exec
 solana-validator ..."); otherwise, when logrotate sends its signal to the
 validator, the enclosing script will die and take the validator process with
 it.
+
+## Stake your validator
```

Comment:
i think this section would fit better immediately after vote account creation.  gonna be pretty easy to miss down here